packages/ui/react/src/components/Primitives/Dropdown/Dropdown.tsx

```diff
@@ -313,16 +313,18 @@ const Dropdown = ({
                         activeBackgroundColor={option.activeBackgroundColor}
                         brokenSequence={option.brokenSequence}
                         onClick={(event) => {
-                          event.preventDefault()
-                          event.stopPropagation()
+                          if(!option.isDisabled) {
+                            event.preventDefault()
+                            event.stopPropagation()
 
-                          if (option.onClick) {
-                            option.onClick()
-                          } else if (onSelect) {
-                            onSelect(option)
-                          }
+                            if (option.onClick) {
+                              option.onClick()
+                            } else if (onSelect) {
+                              onSelect(option)
+                            }
 
-                          popupState.close()
+                            popupState.close()
+                          }
```

Comment:
I believe we can avoid the extra indentation using an early return [1]

This helps whoever reads to follow the happy path by following the same indentation

Can you apply the suggestion below?

[1] https://medium.com/swlh/return-early-pattern-3d18a41bba8

This source was mentioned in multiple websites during my search for this pattern name, if you are interested
```suggestion
                          if(option.isDisabled) {
                            return
                          }

                          event.preventDefault()
                          event.stopPropagation()

                          if (option.onClick) {
                            option.onClick()
                          } else if (onSelect) {
                            onSelect(option)
                          }

                          popupState.close()
```

Reply:
Suggestion added.